Source/WebCore/page/AlternativeTextClient.h

```diff
@@ -38,11 +38,11 @@ enum ReasonForDismissingAlternativeText {
     ReasonForDismissingAlternativeTextAccepted
 };
 
-enum AlternativeTextType {
+enum class AlternativeTextType : uint8_t {
     AlternativeTextTypeCorrection = 0,
     AlternativeTextTypeReversion,
     AlternativeTextTypeSpellingSuggestions,
-    AlternativeTextTypeDictationAlternatives
+    DictationAlternatives
```

Comment:
It seems a good idea to rename AlternativeTextTypeDictationAlternatives to DictationAlternatives.
Can we do the same for the others 3 as well now?

Reply:
Added changes

Reply:
Looks good now